flax/linen/module.py

```diff
@@ -1657,10 +1657,11 @@ def __call__(self, x):
 _ParentType = Union[Type[Module], Type[Scope], Type[_Sentinel], None]
 
 def merge_param(name: str, a: Optional[T], b: Optional[T]) -> T:
-  """Merges construction and call time argument.
+  """Merges construction-and call-time argument.
 
-  This is a utility for supporting the pattern where a Module hyper parameter
-  can be passed to ``__init__`` or ``__call__``.
+  This is a utility for supporting the pattern where a Module hyperparameter
```

Comment:
Nit: Perhaps we should say "a pattern" 🤷 

```suggestion
  This is a utility for supporting a pattern where a Module hyperparameter
```